packages/delivery-node/delivery.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Delivery } from '@bugsnag/core/client'
```

Comment:
This works but I don't think is ideal. It's easy enough to use the internal types within the core package but externall we either need to export them or do this. I'm inclined to prefer exporting the `Delivery` type publically but this works for now.

Reply:
discussed elsewhere and agreed this is fine